### DISCUSSION TITLE
New typescript workspace checker not working for next 13

### DISCUSSION THREAD

**Discussion by DawnMD** [vercel/next.js#42758] [Help]
When setting up a new project with `npx create-next-app@latest --experimental-app` and after setting the typescript to use the workspace version there is no error or warning when importing `useState` in server components. Followed the [beta docs](https://beta.nextjs.org/docs/configuring/typescript#using-the-typescript-plugin). Anything I'm doing wrong here?

System: Windows 11
VSCode: 1.73.1
 
![image](https://user-images.githubusercontent.com/34785917/201194794-91b354d5-5659-4210-b01a-1cd61507551a.png)

**Comment by SoaresMG** [comment]
I'm using the new appDir and the same happens to me.
However I'm using macOS 12.2 and latest vscode
